examples/sensortag.py

```diff
@@ -137,7 +137,8 @@ async def run(address, debug=False):
         battery_level = await client.read_gatt_char(BATTERY_LEVEL_UUID)
         print("Battery Level: {0}%".format(int(battery_level[0])))
 
-        def keypress_handler(sender, data):
+        async def keypress_handler(sender, data):
+            asyncio.get_event_loop().call_soon_threadsafe(lambda x: print(x), "Async!")
```

Comment:
This line seems odd. We are already in an async function, so why starting another async task from here? And why the need for threadsafe? We are already on the event loop thread.

Reply:
Yes, I agree, it is odd. I got wrapped up in the async world. What I really wanted to do was call some async method, don't remeber why I did this instead... It would have sufficed with just printing something...

Reply:
Made the calback async only. It is enough.